JitsiConference.js

```diff
@@ -1340,8 +1340,11 @@ JitsiConference.prototype.onIncomingCall
 
     // TODO: remove deprecated cross region property from this specific event
     // once all existing deployments include analytics changes
-    if (window.jitsiDeploymentInfo) {
-        crossRegion = window.jitsiDeploymentInfo.CrossRegion;
+    if (this.options.config.deploymentInfo) {
```

Comment:
Added checks (it's a bit ugly, let me know if you see a cleaner way to do it), and changed it to use the crossRegion property directly.